client/src/components/auth/SignOut.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import * as actions from '../../actions';
+
+class SignOut extends Component {
+
+  componentWillMount() {
+    this.props.signoutUser();
```

Comment:
Do we even need this component? can we not just redirect to splash page after performing all the signout actions?